pylib/openrazer/client/fx.py

```diff
@@ -157,6 +157,27 @@ def wave(self, direction: int) -> bool:
             return True
         return False
 
+    def wheel(self, direction: int) -> bool:
+        """
+        Wheel effect
+
+        :param direction: Wheel direction either WAVE_RIGHT (0x01) or WAVE_LEFT (0x02)
+        :type direction: int
+
+        :return: True if success, False otherwise
+        :rtype: bool
+
+        :raises ValueError: If direction is invalid
+        """
+        if direction not in (c.WAVE_LEFT, c.WAVE_RIGHT):
+            raise ValueError("Direction must be WAVE_RIGHT (0x01) or WAVE_LEFT (0x02)")
```

Comment:
Not sure if it's a good idea to use WAVE_DIRS now and cement this into the basically public API. Could you add WHEEL_LEFT / WHEEL_RIGHT please?

Reply:
Sure, I will add these